Community calendar added to community page

nathenf opened the pull request:
- Added a view of the calendar to community page
- Included a link to subscribe to the community calendar

#856

nathenf committed:
Community calendar added to community page

nathenf commented:
Hey @olayway I've tried to integrate google calendar into this page and it builds but when I check the preview I get: 

_Application error: a client-side exception has occurred (see the browser console for more information)._ 

I can't figure out why or what this means. Could you assist? 🙏

rufuspollock commented on content/community.md:
```diff
@@ -7,7 +7,15 @@ created: 2023-08-30
+Check out the calendar below to join specific events, or if you want to have all the Life Itself community events in your calendar you can subscribe to the entire calendar! 🎉
+
+👉 [Subscribe to the community calendar](https://calendar.google.com/calendar/u/0?cid=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
+<iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23f0ca5e&ctz=Africa%2FCeuta&hl=en_GB&showPrint=0&showTitle=0&showCalendars=0&src=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23E4C441" style="border-width:0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```
I suspect this is the source of the problem in that you have to use React style stuff when doing html  ...

Specifically style is tricky and i think you need to do it as a property i.e. like this

```
style={{border-width: "0"}}
```

```
<iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23f0ca5e&ctz=Africa%2FCeuta&hl=en_GB&showPrint=0&showTitle=0&showCalendars=0&src=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23E4C441" style={{border-width: "0"}} width="800" height="600" frameborder="0" scrolling="no"
```

For background see https://github.com/datopian/flowershow/issues/508#issuecomment-1559970929

rufuspollock commented on content/community.md:
```diff
@@ -7,7 +7,15 @@ created: 2023-08-30
+
+Check out the calendar below to join specific events, or if you want to have all the Life Itself community events in your calendar you can subscribe to the entire calendar! 🎉
+
+👉 [Subscribe to the community calendar](https://calendar.google.com/calendar/u/0?cid=Y19kMDY0NjNiYzhiZmU4YWVlNjI1OTkyMTBmNDI3YzMxNzY0YjhkMjFlYmRhZWY5MDhlNTBhMGYyZWE0YTM1ODVlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
```
I'd recommend a blank line after this before the html ...

rufuspollock reviewed (changes requested):
I think i've found a fix for you!

rufuspollock committed:
[community,bugfix][xs]: correct iframe style property for react model (it is mdx).


Getting uninformative error: "Application error: a client-side exception has occurred (see the browser console for more information)."

Suspect this is the fact we are using html style tag rather than react style - see https://github.com/datopian/flowershow/issues/508#issuecomment-1559970929

rufuspollock committed:
[community,bugfix][xs]: pt 2 of correct iframe style property for react model (it is mdx).


camelcase borderWidth plus use single quotes.

rufuspollock commented:
FIXED ... finally. https://embed-calendar-community-cal.lifeitself-org.pages.dev/community

![image](https://github.com/life-itself/community/assets/180658/2e655745-b513-4f0c-a65b-c7943d3008f6)
